cookbook/webpack-encore.rst

```diff
@@ -140,6 +140,10 @@ and run the following command:
 
 Now you should be able to see the outcome in the browser.
 
+.. note::
+
+   When using custom assets with webpack-encore, you need to update the admin build in the public/build/admin directory: :doc:`build-admin-frontend`
+
```

Comment:
Now I understand your request. This is not correct that this is needed and is more a hint the documentation here was not followed or you did run into any cache issues. If you run into that problem that the admin and website build is conflicting, check the changes about the public output directories so that `public/build` is correctly changed to `public/build/website` everywhere it is documented here.

That way the `public/build/website` is for website and `public/build/admin` is for admin and so the admin build is not required.

If you accidently did a build before doing this changes a `git checkout public/build/admin` should restore your files.

Reply:
Okay, i see, i remember i forgot one replacement when refactoring from /build/ to /build/website, so this is why my admin build was not working.

Thanks

Reply:
Just something about git checkout, are the admin build files supposed to be tracked with GIT ?

Reply:
@oprichnik by default our build files are tracked on git. I personally like it this way as you can checkout a project fastly without have the need to rebuild all things until you can work with the project. Some don't like it because they mostly have merge conflicts in them and like it to keep building part of the CI/CD jobs in deployment.

So at the end do what works best for your team, both has its advantages and disadvantages.